sdk/core/azure-core-test/inc/azure/core/test/test_base.hpp

```diff
@@ -105,15 +105,6 @@ namespace Azure { namespace Core { namespace Test {
       return RemovePreffix(updated);
     }
 
-    void SkipTest()
-    {
-      if (!m_wasSkipped)
-      {
-        m_wasSkipped = true;
-        GTEST_SKIP();
-      }
-    }
-
     std::string RemovePreffix(std::string const& src)
```

Comment:
```suggestion
    std::string RemovePrefix(std::string const& src)
```

Reply:
That's not my change though ,

Reply:
Yes, I know, thank you for fixing - it's been in the code nearby, but we probably would not get to it any time soon otherwise.